tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2651,6 +2652,62 @@ describe('scalar card', () => {
       expect(data[0].RUN).toEqual('100 test alias 1/Run1 name');
       expect(data[1].RUN).toEqual('200 test alias 2/Run2 name');
     }));
+
+    it('adds smoothed column header when smoothed is enabled', fakeAsync(() => {
+      store.overrideSelector(selectors.getMetricsScalarSmoothing, 0.8);
+
+      const runToSeries = {
+        run1: [
+          {wallTime: 1, value: 1, step: 10},
+          {wallTime: 2, value: 10, step: 20},
+          {wallTime: 3, value: 20, step: 35},
+        ],
+      };
+      provideMockCardRunToSeriesData(
+        selectSpy,
+        PluginType.SCALARS,
+        'card1',
+        null /* metadataOverride */,
+        runToSeries
+      );
+      store.overrideSelector(
+        selectors.getCurrentRouteRunSelection,
+        new Map([['run1', true]])
+      );
+      store.overrideSelector(getMetricsLinkedTimeSelection, {
+        start: {step: 20},
+        end: null,
+      });
+
+      const fixture = createComponent('card1');
+      fixture.detectChanges();
+      const scalarCardComponent = fixture.debugElement.query(
+        By.directive(ScalarCardComponent)
+      );
+
+      expect(scalarCardComponent.componentInstance.dataHeaders).toContain(
+        ColumnHeaders.SMOOTHED
+      );
+
+      expect(
+        scalarCardComponent.componentInstance.getTimeSelectionTableData()[0]
+          .SMOOTHED
+      ).toBe(scalarCardComponent.componentInstance.dataSeries[1].points[1].y);
```

Comment:
To make it clear and simpler in testing, I think it's a good practice to make the input be a simple number or string instead of variable. (constant variable is fine)
Could you replace the variable here with a number (or string)?

Reply:
Yea it was just a weird number so I thought this might make more sense. I am cool using it though. Changed.

Reply:
If the number is not intuitive, please add comment above to explain. I guess this is caused by values of 1, 20, 30. Yet I cannot recall how is the smooth calculated.